### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sjib**

### CODE REVIEW

**Review by ponceta — APPROVED** [review]
As soon as I have your green light @sjib I'll merge this so we have something at least.

**Comment by sjib** [comment]
@ponceta Great - one point:
 - TWW plugin version
 - TWW datamodel version
 shouldn't they be the same in tww? 2024.0, 20204.1 etc.
Do we still have to distinguish that?

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: Who is that? Contact person in user group? Do we still need that? Or what would be helpful here?

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]: Same question here

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:56]: Maybe just set 
- [ ] specification exist
- [ ] offers handed in 
- [ ] offer accepted
or we completely take this out here.

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:62]: see offers

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (ponceta) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: Applicant -> provider
Developer -> developer
Client -> end user

**Comment by sjib** [comment]
@ponceta Lets start like this

**Pull request merged**
